app.rb

```diff
@@ -14,6 +14,10 @@ class App < Sinatra::Base
 
   configure do
     set :log_level, Logger::DEBUG
+    
+    set :gateways, {
+      send_sms: WifiUser::Gateway::GovNotifySMS.new(ENV.fetch('NOTIFY_API_KEY'))
+    }
```

Comment:
Looks like this is moving quite quickly towards a usecase factory.
While I'm not against this, we've not done this on any other application.

Wonder if we should defer this until it becomes a pain to manage the dependencies.

Reply:
I get the concern.
This is to avoid adding more responsibility to the controller. Given that it's still within the file, and is more or less the same as how we've been doing it elsewhere (it's just assigned to a context variable instead of a scoped variable), then this should be fine